migrate/v0_14/migrate.go

```diff
@@ -32,9 +32,9 @@ import (
 )
 
 var (
-	GenesisTime   = time.Date(2021, 3, 4, 15, 0, 0, 0, time.UTC)
-	RewardEndTime = time.Date(2022, 2, 25, 14, 0, 0, 0, time.UTC)
-	ClaimEndTime  = time.Date(2026, 2, 25, 14, 0, 0, 0, time.UTC)
+	GenesisTime   = time.Date(2021, 3, 24, 15, 0, 0, 0, time.UTC)
```

Comment:
Why was the duration from genesis time to reward time lengthened by a week? Was 51 weeks and now it's 52.

Reply:
Typo in original - reward rates should have been stable for 52 weeks. A minor point anyway, as they will be updated by governance before then.